index.html

```diff
@@ -428,6 +437,14 @@ <h2 id="setup">Setup</h2>
     a bit difficult, so we recommend an all-in-one installer.
   </p>
 
+  <p>
+    We will teach Python using the IPython notebook, a programming environment
+    that runs in a web browser. For this to work you will need a reasonably
+    up-to-date browser. The current versions of the Chrome, Safari and
+    Firefox browsers are all supported (some older browsers, including Internet Explorer
+    version 9 and below, are not).
+  </p>
+
```

Comment:
We don't always teach with the Notebook (in fact, I never do any more with novices - too many new things at once).